script/Arcueid/Day 5/05_00_ARC05_9B.txt

```diff
@@ -251,22 +267,27 @@ You would freeze to death in two seconds if you got within ten meters of him.
 [sha:730cd12c4670d0fe2937138f0711e42e0b764633]{
 -- Page 622, Offset 10310.
 -- このままではいずれ俺も倒れる。雪山で遭難した人間が眠ってしまうのと同じだ。人の場合、体温が摂氏34度以下、あるいは摂氏43度以上で脳細胞が停止し、意識が消失してしまう。
-At this rate, I'll eventually fall as well. It's no different to how people struck by disaster in the snow-capped mountains eventually succumb to sleep. When a human's body temperature is below 34℃ or above 43℃, their brain cells shut down and they fall unconscious.
+// 遭難した人間 is "disaster-ed humans" but taking it closer to the meaning in
+// 「遭難信号」 and 「遭難救助隊」 to just be "trapped" in this context.
+At this rate, I'll eventually fall as well.
+ It's the same as humans trapped in the snow-capped mountains succumbing to sleep.
```

Comment:
trapped slightly implies agency (trapped _by_ someone) to me, maybe stranded or something instead?